src/Http/Http.Extensions/gen/StaticRouteHandlerModel/StaticRouteHandlerModel.Emitter.cs

```diff
@@ -15,11 +15,11 @@ public static string EmitHandlerDelegateType(this Endpoint endpoint)
     {
         if (endpoint.Parameters.Length == 0)
         {
-            return endpoint.Response == null || endpoint.Response.IsVoid ? "System.Action" : $"System.Func<{endpoint.Response.WrappedResponseType}>";
```

Comment:
EmitHandlerDelegateCast and EmitHandlerDelegateType are mostly the same, the main difference I see is one uses explicit namespaces and one handles nullable for the display name. Can we merge these two methods?

Reply:
Yep, the other distinction is that one emits `System.Func` and the other emits `Func` because they are generated in namespaces with different usings but I think it is fine to use `System.Func` everywhere to avoid code duplication.